### PR TITLE
Refactor/68 toast helper

### PR DESCRIPTION
**Pull request by tanmaythole**
Created toast helper

### CODE REVIEW

File: src/utils/apisErrorMessage.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+// helper function to filter backend apis error message if api fails
+export const apisErrorMessage = (err) => {
+  const {
+    data: { error },
+  } = err;
+  const commonErr = "Something Went Wrong!";
```
Comment (ChinmaySonawane) [inline src/utils/apisErrorMessage.js]: we can move this into constant

**Review by ChinmaySonawane — APPROVED** [review]

File: src/utils/apisErrorMessage.js (hunk 1 above)
Comment (tanmaythole) [reply src/utils/apisErrorMessage.js]: done 👍🏻

File: src/constants/userMessagesConstants.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
+export const LOGOUT_SUCCESS = "Logout successfully!";
```
Comment (ChinmaySonawane) [inline src/constants/userMessagesConstants.js]: add comments

File: src/constants/userMessagesConstants.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+};
+
+export const TESTSUITES = {
+  ADD_NEW_SUCCESS: "Testsuite added successfully!",
+  UPDATE_SUCCESS: "Testsuite updated successfully!",
+};
+
+export const MEMBERS = {
+  ADD_NEW_SUCCESS: "Members added successfully!",
+  REMOVE_SUCCESS: "Member Removed Successfully! ",
```
Comment (ChinmaySonawane) [inline src/constants/userMessagesConstants.js]: consistency in the user messages and errors

File: src/store/SuperAdmin/sagas.js [hunk 4]
```diff
@@ -1,19 +1,26 @@
...
-        toast.error(error.data.error)
-    }
+  try {
+    yield call(addAdminsApi, payload);
+    toastMessage(ADMINS_ADD_SUCCESS);
+    yield put(addAdminsSuccess());
+  } catch (error) {
+    const errorMessage = apisErrorMessage(error);
+    yield put(addAdminsFailure());
+    toastMessage(errorMessage, "danger");
```
Comment (ChinmaySonawane) [inline src/store/SuperAdmin/sagas.js]: use "error" instead of "danger"

Comment (tanmaythole) [reply src/store/SuperAdmin/sagas.js]: Done 👍🏻

File: src/constants/userMessagesConstants.js (hunk 3 above)
Comment (tanmaythole) [reply src/constants/userMessagesConstants.js]: Done 👍🏻

File: src/constants/userMessagesConstants.js (hunk 2 above)
Comment (tanmaythole) [reply src/constants/userMessagesConstants.js]: Done 👍🏻

**Review by ChinmaySonawane — APPROVED** [review]

**Pull request merged**